### ISSUE TITLE
File does not seem to be in correct format

### ISSUE THREAD

**Issue by mkhademi** [jvgemert/apt#1]
Hi, 

I have successfully installed all the dependencies on my system. After running your ./doExample.sh, I get the following error: "('File does not seem to be in correct format: %s' % featGzipFname)"

I carefully followed all the instructions. Could you tell me what could go wrong? Is it possible that the version of my h5py installation is different than yours and caused the following error?

Thanks a lot for your help!

`./doExample.sh
*\* Step 0: try to compile parts to cython **
Done compiling

*\* Step 1: Extract trajectories from the video **
run tubePy/extractDT.sh example/vid/001.avi example/feat/001/idt.hdf5
Saved temporary trajectories to: ./localexample/vid/001.avi/DenseTrackStab.txt.gz

*\* Convert the trajectories to the format as used for DTP **
python ./tubePy/runConvertDenseTraj2fvec.py ./localexample/vid/001.avi/DenseTrackStab.txt.gz example/feat/001/idt.hdf5
Traceback (most recent call last):
  File "./tubePy/runConvertDenseTraj2fvec.py", line 193, in <module>
    main(featGzipFname, outPath)
  File "./tubePy/runConvertDenseTraj2fvec.py", line 116, in main
    raise ValueError('File does not seem to be in correct format: %s' % featGzipFname)
ValueError: File does not seem to be in correct format: ./localexample/vid/001.avi/DenseTrackStab.txt.gz

clean up temporary file; rm  ./localexample/vid/001.avi/DenseTrackStab.txt.gz`

**Comment by jvgemert** [comment]
Hi,

This error indicates that the IDT format is not as expected. Did you use your own download of the IDT code? Or did you use the one included in 'third'? (I had to change the original IDT code slightly to also output the position)

Can you manually run the command to create the idt file?
ie: "tubePy/extractDT.sh example/vid/001.avi example/feat/001/idt.hdf5"
And then check if this file exists: "./localexample/vid/001.avi/DenseTrackStab.txt.gz" ?
If you unpack it, does it look ok? Ie: are there many trajectories and features there?

Cheers,
~J

**Comment by mkhademi** [comment]
Thanks for your prompt answer! Yes, you are right, I am using my own download of IDT because when I used your version (either the pre-compiled DenseTrackStab or compiled from your improved_trajectory_APT.tar.gz in folder "third"), I get the following error:

*\* Step 1: Extract trajectories from the video **
run tubePy/extractDT.sh example/vid/001.avi example/feat/001/idt.hdf5
./third/DenseTrackStab: symbol lookup error: ./third/DenseTrackStab: undefined symbol: _ZNK2cv9Feature2D7computeERKNS_3MatERSt6vectorINS_8KeyPointESaIS5_EERS1_
Saved temporary trajectories to: ./localexample/vid/001.avi/DenseTrackStab.txt.gz

*\* Convert the trajectories to the format as used for DTP **
python ./tubePy/runConvertDenseTraj2fvec.py ./localexample/vid/001.avi/DenseTrackStab.txt.gz example/feat/001/idt.hdf5
Traceback (most recent call last):
  File "./tubePy/runConvertDenseTraj2fvec.py", line 193, in <module>
    main(featGzipFname, outPath)
  File "./tubePy/runConvertDenseTraj2fvec.py", line 116, in main
    raise ValueError('File does not seem to be in correct format: %s' % featGzipFname)
ValueError: File does not seem to be in correct format: ./localexample/vid/001.avi/DenseTrackStab.txt.gz

Using my own IDT, I wont see the first symbol lookup error but I still get the wrong format error. And in case of using my own IDT, yes, this file exists: "./localexample/vid/001.avi/DenseTrackStab.txt.gz". Here is a snapshot of that, which I believe is in correct format, right?

<img width="558" alt="untitled" src="https://cloud.githubusercontent.com/assets/1362108/14647381/58900f44-0612-11e6-96c7-e07cec7dd94e.png">

Btw, if it helps, based on IDT recommendation here: https://lear.inrialpes.fr/people/wang/improved_trajectories, I'm using OpenCV-2.4.2 and ffmpeg-0.11.1.

Thanks!
Maryam

Update-1: I used your modified DenseTrackStab.cpp only and compiled it with my other IDT downloaded files to generate DenseTrackStab: I am not getting the symbol lookup error but it still complains from wrong format! :(

Update-2: I figured it out. This following line was missing: 
video size, length: 55, width: 720, height: 404
and because of that, it was giving the incorrect format error. I checked your DenseTrackStab.cpp code. Apparently there was a 'tab' missing where you printed this line printf("video size, length: %d, width: %d, height: %d\n", seqInfo.length, seqInfo.width, seqInfo.height); and because of that the output was missing that line!

The rest of the steps worked fine for me.

-Maryam
